script.js

```diff
@@ -0,0 +1,77 @@
+const booksContainer = document.querySelector('.books-container');
+const form = document.querySelector('form');
+
+let books = [
+  {
+    title: 'First Book',
+    author: 'foo',
+  },
+  {
+    title: 'Second Book',
+    author: 'Testroo testyy',
+  },
+];
+
+function generateBooks({ title, author }) {
+  return `<ul>
+        <li>${title}</li>
+        <li>${author}</li>
+        <button>Remove</button>
+    </ul><hr>`;
+}
+
+let booksFromLocalStorage = JSON.parse(localStorage.getItem('books'));
+
+if (booksFromLocalStorage) {
+  booksContainer.innerHTML = booksFromLocalStorage
+    .map((book) => generateBooks(book))
+    .join('');
+} else {
+  booksContainer.innerHTML = books.map((book) => generateBooks(book)).join('');
+}
+
+// booksContainer.innerHTML = books.map(book => generateBooks(book)).join('');
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisy, It is always a good practice to keep your code clean from such comments. :kissing_heart::ok_hand: